### ISSUE TITLE
Blank space in argument description causes failure of dot export

### ISSUE THREAD

**Issue by ggbetz** [christianvoigt/argdown#15]
In the dot-export, argument `<Capability control ineffective 1>` is not renedered correctly. The only difference to the correctly rendered `<Capability control ineffective 2>` is that the blank space after the commata in `(cf. Chapters 9, 12, 13)` is missing.

```
<Control Problem Argument>: Motivation selection is the only control method
that has a chance to avoid the doomsday take-over scenario (cf. Chapter 9).

<Capability control ineffective 1>: Capability control is ineffective and unrelaible 
for superintelligent systems, because the SI is likely to see ways to manipulate 
the world which we don't (cf. Chapters 9,12,13).
  +> <Control Problem Argument>

<Capability control ineffective 2>: Capability control is ineffective and unrelaible 
for superintelligent systems, because the SI is likely to see ways to manipulate 
the world which we don't (cf. Chapters 9, 12, 13).
    +> <Control Problem Argument>
```

![test-0009](https://cloud.githubusercontent.com/assets/3662782/25124169/3f231500-242b-11e7-82ef-35e6e1435ff8.jpg)

**Comment by ggbetz** [comment]
Im Browser sieht der HTML-Export gut aus, aber es wird keine Karte (SVG) erzeugt.

**Comment by christianvoigt** [comment]
fixed in christianvoigt/argdown-map-maker@444413b
